contracts/abstract/PrizeSplit.sol

```diff
@@ -139,6 +139,19 @@ abstract contract PrizeSplit is OwnableUpgradeable {
       return (amount * percentage).div(1000);
   }
 
+  /**
+  * @notice Require the prize split percentage amount does not exceed 100%.
+  * @dev Require the prize split percentage amount does not exceed 100% with the current prize splits.
+  */
+  function _totalPrizeSplitPercentageAmount() internal view returns (uint256) {
+    uint256 _tempTotalPercentage;
+    for (uint8 index = 0; index < _prizeSplits.length; index++) {
```

Comment:
What is the maximum number of splits possible? Would be good to check with a test, maybe a unit test, when this function would break because gas consumption would be too high.